src/functions/retroGen.ts

```diff
@@ -1,21 +1,19 @@
-import { unmarshall } from "@aws-sdk/util-dynamodb";
-import { LambdaClient } from "@aws-sdk/client-lambda";
+import {unmarshall} from "@aws-sdk/util-dynamodb";
```

Comment:
Just a general comment but the formatting of imports across the repo seems is consistent, we usually do the following ...
`import { SomeImport } from "..";`

opposed to 

`import {SomeImport} from "..";`